src/dbus_api/pool.rs

```diff
@@ -287,7 +262,7 @@ fn get_pool_property<F>(i: &mut IterAppend,
                         p: &PropInfo<MTFn<TData>, TData>,
                         getter: F)
                         -> Result<(), MethodErr>
-    where F: Fn(&Pool) -> Result<MessageItem, MethodErr>
+    where F: Fn(&Pool) -> Result<String, MethodErr>
```

Comment:
```String``` type in ```Result``` should be a type parameter, like in ```get_filesystem_property```.

Reply:
ok